src/realtime.c

```diff
@@ -56,22 +56,20 @@ G_DEFINE_TYPE_WITH_CODE (Realtime, realtime, XDP_DBUS_TYPE_REALTIME_SKELETON,
                                                 realtime_iface_init));
 
 static gboolean
-map_pid_if_needed (XdpAppInfo *app_info, pid_t *pid, pid_t *tid, GError **error)
+map_pid (XdpAppInfo *app_info, pid_t *pid, pid_t *tid, GError **error)
 {
-  if (!xdp_app_info_is_host (app_info))
+  if (!xdp_app_info_map_pids (app_info, pid, 1, error))
   {
```

Comment:
Could clean up the messed up indentation while at it.

Reply:
done